python/ray/_raylet.pyx

```diff
@@ -1459,6 +1464,7 @@ cdef class CoreWorker:
                 f"{python_scheduling_strategy}. "
                 f"Valid values are [\"DEFAULT\""
                 f" | DefaultSchedulingStrategy"
+                f" | \"SPREAD\" | SpreadSchedulingStrategy"
```

Comment:
How about we remove the DefaultSchedulingStrategy and SpreadSchedulingStrategy Python classes? Seems they don't really serve a purpose when we have the string form, and we don't accept arbitrary subclasses of SchedulingStrategy.